project4/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>project4</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Francois+One|Josefin+Sans:700|Scope+One" rel="stylesheet">
+</head>
+
+<body>
+    <!-- nav and header -->
+    <div class="the_official_angelique_site_wrapper">
+        <header>
+            <h1 class="tracking-in-expand">APDC<span>2019</span></h1>
+            <nav>
+                <ul>
+                    <li class="nav_button_red"><a class="nav_link" href="#mission">Mission</a></li>
+                    <li class="nav_button_blue"><a class="nav_link" href="#speakers">Speakers</a></li>
+                    <li class="nav_button_red"><a class="nav_link" href="#events">Events</a></li>
+                    <li class="nav_button_blue"><a class="nav_link" href="#tickets">Tickets</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <!-- site banner -->
+            <div class="banner color-change-2x">
+                <a class="ticket_link" href="#tickets">
+                    <h2>The American Political Design Conference 2019</h2>
+                    <h2>June 1st-3rd @ Walter E. Washington Convention Center</h2>
+                </a>
+            </div>
+
+            <!-- intro -->
+            <section class="right_border">
+                <img src="images\asset5.png"
+                    alt="A photogrid of the various different campaign logos from the last 30 years">
+                <p>APDC2019 is an annual design conference held to explore and educate designers about the role,
+                    practices, and most important evolution of political design. The subjects range from the graphic
+                    design of the campaigns to the creative tactics used in message and positioning of candidates,
+                    propositions, and referendums.
+                </p>
+            </section>
+
+            <!-- mission -->
+            <h3 id="mission">Mission</h3>
+            <section class="left_border">
+                <p>Since its inception, the mission of the APDC is to educate designers on how to reach people
+                    regardless of color, creed, or political ideology. With that in mind, any one in attendance of APDC
+                    must respect the political ideology of any other attendees, no matter their differences.Leave your
+                    party
+                    at the door and remember: We are all here to educate and connect on the basis of accurate and
+                    informative
+                    communications.</p>
+                <img src="images\asset4.png"
```

Comment:
```suggestion
                <img src="images/asset4.png"
```
![the other slash](https://media.giphy.com/media/zdg7ZNmbk9ZD2/giphy.gif)